src/AlgAss/Types.jl

```diff
@@ -130,6 +130,13 @@ end
 @attributes mutable struct GroupAlgebra{T, S, R} <: AbstractAssociativeAlgebra{T}
   base_ring::Ring
   group::S
+  # We represent elements using a coefficient vector,
```

Comment:
```suggestion
  # If `sparse` is false we represent elements using a coefficient vector,
```

Reply:
I mean vector in the mathematical sense. They comment there is valid both for sparse and non-sparse representations.